WordPress/Classes/ViewRelated/Stats/Period Stats/Countries/CountriesCell.swift

```diff
@@ -13,6 +13,8 @@ class CountriesCell: UITableViewCell, NibLoadable {
     @IBOutlet weak var subtitlesStackViewTopConstraint: NSLayoutConstraint!
     @IBOutlet weak var rowsStackViewTopConstraint: NSLayoutConstraint!
 
+    @IBOutlet private var topSeparatorLineHeightConstraint: NSLayoutConstraint!
```

Comment:
Hey @danielebogo . I know I approved this, but I just noticed this. Just wondering - why is this `private var` instead of `weak var`?

Reply:
@ScoutHarris `private` because the property doesn't need to be exposed and accessible from outside its class. Also this is not `weak` because the all subviews should be strongly retained. It won't generate any retain cycle. This ensures all the subviews will exist until its parent is deallocated. Having a weak var implicitly unwrapped it's also dangerous because it can be accessed while its status is nil and cause a crash. Apple says _the only time you really need to make an outlet weak is if you have a custom view that references something back up the view hierarchy and in general that's not recommended_.

Reply:
> Apple says the only time you really need to make an outlet weak is if you have a custom view that references something back up the view hierarchy and in general that's not recommended.

Oh, well, it's a good thing Xcode defaults to `weak` when an outlet is created. 🤦‍♂

Reply:
Ah I think XCode keeps the last _configuration_ you used when an outlet is added. I had the same problem IBAction and IBOutlet

Reply:
TIL.